README.md

```diff
@@ -4,13 +4,11 @@ Welcome to the Klipper project!
 
 https://www.klipper3d.org/
 
-Klipper is a 3d-Printer firmware. It combines the power of a general
```

Comment:
3D as per https://en.wikipedia.org/wiki/3D_printing
"a firmware" is wrong, but it could be a firmware _solution_ etc.